### DISCUSSION TITLE
render non-sRGB etc2 textures

### DISCUSSION THREAD

**Discussion by sannyun** [gfx-rs/wgpu#3815] [Q&A] [answered]
I'm trying to render a non-sRGB etc2 texture with GLES backend on android. The result looks darker than expected. It seems like this line caused the problem.
https://github.com/gfx-rs/wgpu/blob/e6314c407048ae20a49c6cba0730949d5bb1a7e8/wgpu-hal/src/gles/conv.rs#L114
What is the purpose of choosing glow::COMPRESSED_SRGB8_ALPHA8_ETC2_EAC instead of glow::COMPRESSED_RGBA8_ETC2_EAC in here?

**Comment by cwfitzgerald** [comment, marked as answer]
This feels like ajust a bug that should be fixed.
